src/EditorFeatures/Core/Implementation/RenameTracking/RenameTrackingTaggerProvider.cs

```diff
@@ -172,10 +172,10 @@ internal static bool WaitForIsRenamableIdentifier(Task<TriggerIdentifierKind> is
             {
                 return isRenamableIdentifierTask.WaitAndGetResult(cancellationToken) != TriggerIdentifierKind.NotRenamable;
             }
-            catch (AggregateException e) when (e.InnerException is OperationCanceledException)
+            catch (TaskCanceledException)
```

Comment:
It's to differentiate between `WaitForIsRenamableIdentifier(canceledTask, CancellationToken.None)` and `WaitForIsRenamableIdentifier(unfinishedTask, new CancellationToken(canceled: true))`. [According to the tests](https://github.com/dotnet/roslyn/blob/9205e7f/src/EditorFeatures/Test/RenameTracking/RenameTrackingTaggerProviderTests.cs#L854-L857), the former should return `false`, while the latter should throw OCE. If I just caught OCE here, the second test would fail.

I don't know why it's designed this way, but I thought I should keep the existing behavior and this does that.

Though there is one case I (or the tests) didn't consider: if the task was faulted with OCE. In that case, I think the method should return `false`, so I have switched to catching OCE and checking its `CancellationToken`.